mirage/views/subjects-acceptable.ts

```diff
@@ -0,0 +1,46 @@
+import { HandlerContext, ModelInstance, Request, Schema } from 'ember-cli-mirage';
+import Subject from 'ember-osf-web/models/subject';
+import { process } from './utils';
+
+function getFilterOpts(
+    queryParams: { [key: string]: string },
+): { type: string, value: string } {
+    if ('filter[parent]' in queryParams) {
+        const { 'filter[parent]': value } = queryParams;
+        return { type: 'parent', value };
+    }
+    const { 'filter[text]': text } = queryParams;
+    return { type: 'text', value: text };
+}
```

Comment:
You could either move `getSubjectsAcceptable` into `mirage/views/provider-subjects.ts` or you could export this function from `provider-subjects.ts` and import it from there so you don't need to duplicate this code.